example/ski-def.json

```diff
@@ -10,7 +10,8 @@
     "taper": 1.4,
     "nose shape":
     {
-      "type": "Basic Bezier",
+      "type": "Flat",
```

Comment:
This is great for testing. For the final commit, let's leave the nose rounded to serve as an example for a normal ski for people. Maybe we should add a few test configs with some of these good test cases?